docs/User/Swarm_Learning_Component_Interactions_using_Reverse_Proxy.md

```diff
@@ -0,0 +1,28 @@
+# Swarm Learning Component Interactions using Reverse Proxy 
+
+A Reverse Proxy Server is a type of proxy server that presents behind the firewall in a private network and directs client requests to appropriate backend or origin servers. Reverse Proxy for Swarm Learning is used to minimize the number of network ports that must be opened for communication between various nodes. An Open port refers to a TCP or UDP port number that is configured to accept packets.
```

Comment:
Rephrase "communication between various nodes" to "communication between various host machines"